### DISCUSSION TITLE
How are we actually meant to use responsive font sizes?

### DISCUSSION THREAD

**Discussion by jeffmarshall** [twbs/bootstrap#30887] [General]
Nothing changes when I resize my viewport with `$enable-responsive-font-sizes: true;`

I get that we should use `rfs(size)` for font-sizes in custom styles, but font sizes don't change when we scale down. How do we take advantage of this system of relative sizes.

**Comment by muhaimincs** [comment]
agree with you. How exactly should we consume this feature? should we install package RFS?
